### PR TITLE
[BXMSPROD-1333] kie-wb-distributions test failures fixed

### PR DESCRIPTION
**Pull request by Ginxo**
**Thank you for submitting this pull request**

**JIRA**: https://issues.redhat.com/browse/BXMSPROD-1333

**referenced Pull Requests**: _(please edit the URLs of referenced pullrequests if they exist)_

* https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/1647
* https://github.com/kiegroup/kie-wb-distributions/pull/1104
* https://gitlab.cee.redhat.com/business-automation/eng-jenkins/-/merge_requests/54

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>a pull request</b> please add comment: <b>Jenkins retest this</b>
 
* <b>a full downstream build</b> please add comment: <b>Jenkins run fdb</b>
  
* <b>a compile downstream build</b> please  add comment: <b>Jenkins run cdb</b>

* <b>a full production downstream build</b> please add comment: <b>Jenkins execute product fdb</b>

* <b>an upstream build</b> please add comment: <b>Jenkins run upstream</b>
</details>

### CODE REVIEW

File: .github/workflows/pull_request.yml [hunk 1]
```diff
@@ -64,9 +64,14 @@ jobs:
           path: ~/.m2
           key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
           restore-keys: ${{ runner.os }}-m2
+      - name: Install firefox and set FIREFOX_FOLDER env variable
+        run: |
+          wget https://ftp.mozilla.org/pub/firefox/releases/60.9.0esr/linux-x86_64/en-US/firefox-60.9.0esr.tar.bz2
+          tar -xf firefox-60.9.0esr.tar.bz2
+          echo "FIREFOX_FOLDER=`pwd`/firefox" >> $GITHUB_ENV
```
Comment (winklerm) [inline .github/workflows/pull_request.yml:71]: Is this correct?

**Review by winklerm — COMMENTED** [review]
One question, otherwise ok.

File: .github/workflows/pull_request.yml (hunk 1 above)
Comment (winklerm) [reply .github/workflows/pull_request.yml:71]: I mean setting the FIREFOX_FOLDER var.

**Review by tomasdavidorg — APPROVED** [review]

File: .github/workflows/pull_request.yml (hunk 1 above)
Comment (tomasdavidorg) [reply .github/workflows/pull_request.yml:71]: I think it is correct. `$FIREFOX_FOLDER` is not directly used but `${{ env.FIREFOX_FOLDER }}` is used. And according to this https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#setting-an-environment-variable looks good.

**Review by mareknovotny — APPROVED** [review]

**Pull request merged**